website/docs/components/table/advanced-table/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,42 @@
+## Anatomy
+
+### Advanced Table headers
+
+![](/assets/components/table/advanced-table/advanced-table-header-select-anatomy.png)
+
+| Element          | Usage                                           |
+|------------------|-------------------------------------------------|
+| Checkbox         | Optional, but required when cells yield a checkbox |
+| Label            | Required                                        |
+| Tooltip button   | Optional                                        |
+| Sort button      | Options: none, ascending, descending            |
+| Container        | Required                                        |
+
+### Advanced Table cells
+
+![](/assets/components/table/advanced-table/advanced-table-cell-parent-anatomy.png)
+
+| Element      | Usage    |
+|--------------|----------|
+| Expand       | Optional |
+| Cell content | Required |
+| Icon         | Optional |
+| Container    | Required |
+
+![](/assets/components/table/advanced-table/advanced-table-cell-nested-anatomy.png)
+
+| Element      | Usage    |
+|--------------|----------|
+| Indentation  | Required |
```

Comment:
Nit: in the image, its labelled as "Nested" not "indentation". Which is correct?